test/common_utils.py

```diff
@@ -132,25 +124,16 @@ def remove_prefix_suffix(text, prefix, suffix):
 
```

Comment:
The `munged_id` variable in the exact above line should have also been removed.

Reply:
I'll submit a fix